src/wristpy/processing/calibration.py

```diff
@@ -272,7 +281,14 @@ def _calibrate(self, acceleration: models.Measurement) -> LinearTransformation:
             2014 Oct 1;117(7):738-44. doi: 10.1152/japplphysiol.00421.2014.
         """
         no_motion_data = self._extract_no_motion(acceleration=acceleration)
-        linear_transformation = self._closest_point_fit(no_motion_data=no_motion_data)
+        if self.method == "GGIR":
```

Comment:
We should discuss together with @frey-perez whether this is the best method for implementing this, rather than a separate class per calibrator. The reason being that it'd play more nicely with the proposal in #95 to allow custom calibrators into the orchestrator.